File: tools/content-verify/__fixtures__/tracks/demo/problems/lc-9006-in-place/Solution.java
import java.util.*;

class Solution {
    public void reverse(int[] nums) {
        for (int left = 0, right = nums.length - 1; left < right; left++, right--) {
            int swap = nums[left];
            nums[left] = nums[right];
            nums[right] = swap;
        }
    }
}

File: tools/content-verify/__fixtures__/tracks/demo/problems/lc-9006-in-place/solution.py
from typing import List


class Solution:
    def reverse(self, nums: List[int]) -> None:
        left, right = 0, len(nums) - 1
        while left < right:
            nums[left], nums[right] = nums[right], nums[left]
            left += 1
            right -= 1
